otherlibs/win32unix/unixsupport.c

```diff
@@ -170,45 +170,118 @@ void win32_maperr(DWORD errcode)
 }
 
 /* Windows socket errors */
-
+#ifndef EWOULDBLOCK
```

Comment:
You are right, erno defines them to different values. I think the better solution would be to just replace the uses of the macro in this file by the corresponding value.